composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": ">=7.3",
```

Comment:
Do we really use 7.3 features in this package? I would like to have a possibility to use those changes in older packages if such requirements would fit in the future, especially that rebranding introduces a need for forward compatibility.

Reply:
It comes from https://github.com/ezsystems/ezplatform-code-style/pull/10#discussion_r587431167 comment.

> We can bump PHP version to 7.3 as we plan to release major version

I don't mind either way. I'll change it to 7.1 for the moment.